### DISCUSSION TITLE
Embed symbols in static libraries for MSBuild

### DISCUSSION THREAD

**Discussion by sanxiyn** [microsoft/vcpkg#19290] [Ideas]
In #1694 (commit 26516fe485b0e9048dd4809256a7e4526957c6e9), it is reported that "vcpkg now embeds symbols in static libs by default". But the change was only applied to ports using CMake and Meson, not MSBuild, that is, ports using `vcpkg_install_msbuild`.

**Comment by JackBoosY** [comment]
This issue hasn’t been updated in a year; if it is still an issue, please reopen this issue.

**Comment by sanxiyn** [comment]
It is still an issue, so please reopen. Thanks! (I can't reopen myself.)

**Comment by sanxiyn** [comment]
In ad7fa55, the relevant code got extracted so that other build systems can use it.

**Comment by JackBoosY** [comment]
Okay, I'm going to close this issue now.

**Comment by sanxiyn** [comment]
What? Why?

The code got extracted so that build systems like MSBuild can use it, but as far as I can tell, MSBuild is not using it yet...

**Comment by JackBoosY** [comment]
Ops, sorry.
